FutureNorthants/northants-website
# Add navigation image to service & service landing pages

- Add an optional Navigation Image to Service Pages and Service Landing Pages
- Add ability to Display Images in sections
- Add an optional Meta Image in the SEO section

## To Test
- Clone down the branch, then `./drush.wrapper @docker config:import`
- Clear the cache `./drush.wrapper @docker cache:rebuild`
- Edit a Service Page that is in a section of a Service Landing Page, by selecting a Nav image and publish
- Edit the Service Landing Page by checking the 'Display Images' checkbox for the section, then choose a Meta Image in the SEO tab, then publish
- Use the below query with the path to the Service Landing Page and ensure the imageSmall, imageLarge and imageAltText are returned, as well as the metaImage

```graphql
{
  route(path: "/path-to-page") {
    __typename
    ... on DrupalNodeRoute {
      node {
        ... on ServiceLandingPageNode {
          title
          id
          metaImage
          hasSections {
            __typename
            name
            hasImages
            pages {
              __typename
              title
              metaImage
              imageAltText
              imageLarge
              imageSmall
            }
          }
        }
      }
    }
  }
}
```
